src/udisks2monitor.cpp

```diff
@@ -203,6 +203,41 @@ void UDisks2::Monitor::unmount(const QString &devicePath)
     startMountOperation(devicePath, UDISKS2_FILESYSTEM_UNMOUNT, m_blockDevices->objectPath(devicePath), arguments);
 }
 
+void UDisks2::Monitor::setLabel(const QString &devicePath, const QString &label)
+{
+    if (m_blockDevices->find(devicePath)) {
+        QVariantList args;
+        QVariantMap opts;
+        args << opts;
+
+        const QString dbusMethod = UDISKS2_FILESYSTEM_UNMOUNT;
+
+        QDBusInterface udisks2Interface(UDISKS2_SERVICE,
```

Comment:
Please, use `QDBusMessage::createMethodCall()` instead because `QDBusInterface` object can query introspection and block.

Reply:
done